gs_bucket_backup/test_bucket_sync.py

```diff
@@ -0,0 +1,47 @@
+import unittest.mock
+from bucket_sync import GCPBucketBackup
+
+
+class TestBucketSync(object):
```

Comment:
Tests! Oh, my! :smiley:

Reply:
Was not able to run these tests locally, even with Python 3.5.3, due to a missing `pathlib2` import. Once added, tests passed A-OK. Will tack on to the reqs.

Reply:
you used `pipenv run pytest` ? Huh its working in CI and locally for me. I did see this issue early on when I created a local pipenv with python 2.x - let me know what you find out?